### PR TITLE
[FIX] "Idle Time Limit" using milliseconds instead of seconds

### PR DESCRIPTION
**Pull request by kaiiiiiiiii**
I've closed my other PR (#9675) in favor of this one, because I really messed up with almost everything.

This time I forked the latest develop branch and requested to merge the PR back into develop. I'm sorry for any inconvenience, but I'm pretty new to git group projects :)

**Summary old PR**
For users it's pretty unhandy to fill out the "Idle Time Limit" in milliseconds. As discussed with @geekgonecrazy and @JSzaszvari it would be more practical to set this value in seconds. @sampaiodiego targets version 0.63.0 for this fix.

### CODE REVIEW

File: packages/rocketchat-i18n/i18n/en.i18n.json [hunk 1]
```diff
@@ -999,7 +999,7 @@
   "IssueLinks_LinkTemplate": "Template for issue links",
   "IssueLinks_LinkTemplate_Description": "Template for issue links; %s will be replaced by the issue number.",
   "It_works": "It works",
-  "Idle_Time_Limit": "Idle Time Limit",
+  "Idle_Time_Limit": "Idle Time Limit (Sec.)",
```
Comment (graywolf336) [inline packages/rocketchat-i18n/i18n/en.i18n.json]: Can you add a `Idle_Time_Limit_Description` property and describe what it does?

**Comment by kaiiiiiiiii** [comment]
@RocketChat/core Can someone please review this PR? Thanks

**Comment by kaiiiiiiiii** [comment]
@sampaiodiego Can you please add a milestone to this PR?

**Comment by geekgonecrazy** [comment]
@kaiiiiiiiii can you rename your migration file to 109 and change from 108 to 109 inside the file also. Seems someone else added a migration for 108 and its already been merged

**Comment by kaiiiiiiiii** [comment]
@geekgonecrazy Changed migration version number again, as there has been a new merge conflict with another migration.

Cheers

**Pull request merged**